src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/CounterPayload.cs

```diff
@@ -110,7 +64,7 @@ IEnumerator IEnumerable.GetEnumerator()
                 yield return new KeyValuePair<string, object>("IntervalSec", IntervalSec);
                 yield return new KeyValuePair<string, object>("Series", $"Interval={IntervalSec}");
                 yield return new KeyValuePair<string, object>("CounterType", "Mean");
-                yield return new KeyValuePair<string, object>("MetaData", MetaData);
+                yield return new KeyValuePair<string, object>("MetaData", Metadata);
```

Comment:
Should the key "MetaData" match the variable name "Metadata"?

Reply:
They match below.

Reply:
Oops. Yes they should match. Thanks for catching this!